source/conditional-simplify-deep.d.ts

```diff
@@ -0,0 +1,62 @@
+/**
+Recursively simplifies a type while including and/or excluding certain types from being simplified.
```

Comment:
Please preserve the code example like it was originally, this includes empty lines.

Reply:
@romanywu I was talking in general, not just this example. All the code.